modules/nodes-descheduler-configuring-profiles.adoc

```diff
@@ -35,30 +35,33 @@ spec:
   logLevel: Normal
   managementState: Managed
   operatorLogLevel: Normal
+  mode: Predictive                                     <1>
   profileCustomizations:
-    namespaces:                                        <1>
+    namespaces:                                        <2>
       excluded:
       - my-namespace
-    podLifetime: 48h                                   <2>
-    thresholdPriorityClassName: my-priority-class-name <3>
-  profiles:                                            <4>
+    podLifetime: 48h                                   <3>
+    thresholdPriorityClassName: my-priority-class-name <4>
+  profiles:                                            <5>
   - AffinityAndTaints
-  - TopologyAndDuplicates                              <5>
+  - TopologyAndDuplicates                              <6>
   - LifecycleAndUtilization
   - EvictPodsWithLocalStorage
   - EvictPodsWithPVC
 ----
---
```

Comment:
@sabrinajess Hey, it looks like you accidentally broke a block when you removed this line. If you put the `--` back on line 51, your build should stop failing. 🤞🏻 

You can see the related formatting issues here: http://file.rdu.redhat.com/sjess/CNV21088/nodes/scheduling/nodes-descheduler.html#nodes-descheduler-configuring-profiles_nodes-descheduler (please also put this link in your PR description before you request peer review)

More info about blocks (this is an "open" block): https://docs.asciidoctor.org/asciidoc/latest/blocks/build-basic-block/#summary-of-built-in-blocks

Reply:
Thank you for investigating!